File: src/main/java/com/googongill/aditory/external/chatgpt/dto/Choice.java
package com.googongill.aditory.external.chatgpt.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;
import lombok.Getter;

import java.io.Serializable;

@Getter
@Builder
public class Choice implements Serializable {
    private Integer index;
    private Message message;
    @JsonProperty("finish_reason")
    private String finishReason;
}

File: src/main/java/com/googongill/aditory/external/chatgpt/dto/Choice.kt
package com.googongill.aditory.external.chatgpt.dto

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

data class Choice(
    val index: Int,
    val message: Message,
    @JsonProperty("finish_reason")
    val finishReason: String
) : Serializable